README.md

```diff
@@ -1,23 +1,94 @@
-# Onigumo #
+# Onigumo
 
-## About ##
+## O projektu
 
-This is an attempt to build just another web-crawler, also called _Spider_. Its purpose is to get data from a website in a form of a list of objects. This data can be then used to download linked files and place them in a database or a folder structure.
+Onigumo je jeden z dalších web-crawlerů, také známých pod pojmem _Spider_.
+Onigumo obecně řídí toky dat, určuje směry toků a v jaké chvíli se
```

Comment:
This sentence does not make sense to me, sorry.

Reply:
The idea is to tell people generally purpose. I think you told me this sentence when you've been describing the project to me. 😄 Anyway, we can delete it, I have no problem with that.